addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -764,7 +764,7 @@ def _import_fattura_pa(self, tree, invoice):
 
                     # default_res_id is had to context to avoid facturx to import his content
```

Comment:
shall we remove that comment as well ? 

To me it also looks like 'no_new_invoice' is enough
https://github.com/odoo/odoo/blob/f234b4fba40d3c0d7e5b4828dbbd033f6a667ea2/addons/account/models/account_move.py#L2901